spec/integration/posts_index_spec.rb

```diff
@@ -3,11 +3,11 @@
 RSpec.describe 'Posts index page', type: :system do
   describe 'check the content of the index page' do
```

Comment:
- [ ] You are missing tests for the following scenarios in the posts index page.

  - Tests that asserts you can see the number of posts the user has written.
  - Tests that asserts you can see some of the post's body.